Add uninstallation info (#510)
pi-hole/docs: docs/main/uninstall.md

> Uninstallation is as simple as running the following command:

Reviewer: ```suggestion
Uninstalling Pi-hole can be started by running the following command:
```

`pihole uninstall`

!!! warning

>     If your are unsure whether a packag should be removed during uninstallation, we recommend leaving it installed, as usually required system packages are among them!

Reviewer: ```suggestion
    If your are unsure whether a package should be removed during uninstalling, we'd recommend leaving it installed, as required system packages may be among them!
```
